packages/scss/src/components/inputs/textfield/_textfield.framed.scss

```diff
@@ -30,6 +50,7 @@
 
 	.textfield-suffix {
 		bottom: _component("textfield.framed.bottom-padding");
+		position: absolute;
 		right: _component("textfield.framed.side-padding");
 		top: auto;
```

Comment:
based on how it is positionned in the textfield.scss, the top auto might not be necessary

Reply:
✔️